salt/states/service.py

```diff
@@ -349,6 +349,9 @@ def running(name,
     # Check if the service is available
     try:
         if not _available(name, ret):
+            if __opts__['test']:
+                ret['result'] = None
+                ret['comment'] = 'Service {0} would have been started'.format(name)
```

Comment:
Message should probably be, `Service {0} not present; if created in this state run, it would have been started', or something like that.